mozetl/search/daily_search_rollups.py

```diff
@@ -129,6 +130,7 @@ def roll_up_searches(spark, frame):
         "distribution_id",
         "default_provider",
         "search_counts.engine as search_provider",
+        "search_counts.source as search_source",
```

Comment:
A comment about filtering out search sources not related to search engines. An example of ignored source (`filteron%`).